shared/reducers/fs.js

```diff
@@ -67,6 +67,26 @@ export default function(state: Types.State = initialState, action: FsGen.Actions
     case FsGen.download:
     case FsGen.openInFileUI:
       return state
+    case FsGen.fuseStatus:
+      return state
```

Comment:
why not just fallthrough above. we usually have a section of saga-only actions which all fallthrough